### PR TITLE
add log filter test to run after browser tests

### PR DESCRIPTION
**Pull request by will-byrne**
add log filter tests to run after the browser tests.

### CODE REVIEW

File: packages/component-logger/server/log.filter-ci.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const readline = require('readline')
+const fs = require('fs')
+
+const logPath = process.env.XPUB_LOG_PATH || '../..'
+
+describe('CI test for log filtering', () => {
+  const lines = []
+  jest.setTimeout(30000)
```
Comment (diversemix) [inline packages/component-logger/server/log.filter-ci.js]: This interesting!

File: packages/component-submission/server/resolvers/submitManuscript.js [hunk 2]
```diff
@@ -19,6 +19,8 @@ async function submitManuscript(_, { data }, { user, ip }) {
     )
   }
 
+  logger.warn('hello@testing.com')
```
Comment (diversemix) [inline packages/component-submission/server/resolvers/submitManuscript.js]: This is interesting too!

File: scripts/pipeline-log-filter-test.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+echo "Started browser test script"
+set -ex
+echo "set ex"
+cd packages/component-logger
+echo "CD into logger folder"
+find / | grep 'xpub.log'
+npx jest --testMatch=**/log.filter-ci.js
```
Comment (diversemix) [inline scripts/pipeline-log-filter-test.sh]: Should we rename `log.filter-ci.js` to `log.filter-ci.test.js` for consistency?

File: scripts/pipeline-log-filter-test.sh [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+echo "Started browser test script"
+set -ex
+echo "set ex"
+cd packages/component-logger
+echo "CD into logger folder"
```
Comment (diversemix) [inline scripts/pipeline-log-filter-test.sh]: Could change to :
```
echo " logger folder: $(pwd)"
```

**Review by diversemix — APPROVED** [review]

File: scripts/pipeline-log-filter-test.sh (hunk 4 above)
Comment (diversemix) [reply scripts/pipeline-log-filter-test.sh]: @will-byrne ... I thought this went in :thinking:

Comment (will-byrne) [reply scripts/pipeline-log-filter-test.sh]: it has now

**Review by Cooryd — APPROVED** [review]

**Pull request merged**